Library/Styles/Colors.swift

```diff
@@ -179,6 +179,11 @@ extension UIColor {
     return .hex(0x1A8CFF)
   }
 
+  // 0x2b60ff
+  public static var ksr_blue: UIColor {
+    return .hex(0x2b60ff)
```

Comment:
Just a question...was this generated by our `Colors.script`? Looks like the casing is different so just wanted to double check..we would lose these changes next time someone runs the script.